### ISSUE TITLE
Codeowners file is invalid

### ISSUE THREAD

**Issue by derek-ho** [opensearch-project/security-dashboards-plugin#1883]
**What is the bug?**
Seeing this error on CODEOWNERS file:

```
Unknown owner on line 1: make sure @davidlago exists and has write access to the repository
…123 @cwperks @DarshitChanpura @davidlago @derek-ho @peternied @RyanL1997 @scrawfor9
```

We should probably move Dave to emeritus maintainer and remove him from codeowner file. 

**What is the expected behavior?**
No error

**Comment by cwperks** [comment]
Thank you Derek. Do you plan to create a PR to address this issue?

**Comment by derek-ho** [comment]
Yes I will do so later today, I think security backend plugin also needs the same thing. I will create an issue and PR.

**Comment by peternied** [comment]
Let's make sure to get @davidlago's input on his status in the project before making changes

**Comment by peternied** [comment]
> Let's make sure to get @davidlago's input on his status in the project before making changes. IMO there are two potential outcomes, 1) the permissions issue gets resolved and the file starts working again, 2) David chooses to become emeritus and he is removed from the codeowners (and maintainers.md is updated)

- _From https://github.com/opensearch-project/security/issues/4234_

**Comment by derek-ho** [comment]
Yup. Confirmed with @davidlago this move to emeritus maintainer in a private conversation.
